packages/now-cli/test/dev/integration.js

```diff
@@ -1347,3 +1347,15 @@ test(
     await testPath(200, `/api/user.sh`, /Hello, from Bash!/m);
   })
 );
+
+test(
+  '[vercel dev] Should work with nested `tsconfig.json` files',
+  testFixtureStdio(
+    'nested-tsconfig',
+    async testPath => {
+      await testPath(200, `/`, /Nested tsconfig.json test page/);
+      await testPath(200, `/api`, 'Nested `tsconfig.json` API endpoint');
+    },
+    { skipDeploy: true }
```

Comment:
Why skipDeploy? Seems like it should also test production to make sure it works the same as dev?

Reply:
Sure, updated in 5a3544164ceb65e0aa1f3a4d0a930bcba7b41e1e.